examples/opentelemetry/envoy-1.yaml

```diff
@@ -4,6 +4,48 @@
 #   10001 -> routes to `envoy-2`
 
 static_resources:
+  clusters:
```

Comment:
please keep clusters _after_ listeners as that is consistent with the existing codebase

Reply:
But then I cannot use YAML anchors in a non-confusing way :-(

Reply:
Done in 348c1dc0e1

Reply:
Done in 91f0514864232a285b9ec9ef220d00974d4f3705